backend/spring-routie/src/main/java/routie/logging/logger/ClientRequestLogger.java

```diff
@@ -0,0 +1,59 @@
+package routie.logging.logger;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.stereotype.Component;
+import routie.logging.extractor.dto.HandlerParameterDto;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class ClientRequestLogger {
+
+    private final ObjectMapper objectMapper;
+
+    public void log(
+            final String httpMethod,
+            final String url,
+            final String clientIp,
+            final String handlerMethod,
+            final List<HandlerParameterDto> handlerParams,
+            final boolean isSuccess,
+            final long executionTime
+    ) {
+        try {
+            putMdcFields(httpMethod, url, clientIp, handlerMethod, handlerParams, isSuccess, executionTime);
+            log.info("");
```

Comment:
```suggestion
            log.atInfo().log("");
```
이렇게 선언해주면 불필요한 자원 활용을 줄일 수 있습니다. [참고 자료](https://www.slf4j.org/manual.html)

Reply:
한번 조사해보고 적용 적극 고려해보겠습니다 👍

Reply:
거의 info 로그를 찍는 것이 확실하고, 
값을 빌드하지 않고 단순 스트링이거나 없는 현 상황에서는
차니 말대로 `log.atInfo().log("")`를 사용하지 않는 것이 더 나을 수 있을 것 같습니다. 
이후 다른 로깅에서 활용을 고민해봅시다 👍🏻